modules/appeals_api/spec/lib/stats_report_spec.rb

```diff
@@ -26,6 +26,7 @@ def create_transition_groups(group_size: 1, num_groups: described_class::STATUS_
   let(:outside_range_groups) { create_transition_groups }
 
   before do
+    allow(AppealsApi::AppealReceivedJob).to receive(:perform_async)
```

Comment:
Not doing this led to errors from VANotify when it was unable to find the correct template for an email sent when moving an example appeal to the 'submitted' state via `update_status!` in these specs